sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveSessionCatalog.scala

```diff
@@ -69,49 +69,59 @@ private[sql] class HiveSessionCatalog(
     // Current thread context classloader may not be the one loaded the class. Need to switch
     // context classloader to initialize instance properly.
     Utils.withContextClassLoader(clazz.getClassLoader) {
-      Try(super.makeFunctionExpression(name, clazz, input)).getOrElse {
```

Comment:
@AngersZhuuuu, can you get rid of this unrelated diffs?

Reply:
> @AngersZhuuuu, can you get rid of this unrelated diffs?

See  https://github.com/apache/spark/pull/29054#discussion_r452901062
Don't know how can I get the exception without change identation

Reply:
> @AngersZhuuuu, can you get rid of this unrelated diffs?

How about current change, it won't change indentation.